<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleCodeEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="window-close">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Find</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Find Prev</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Find All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="window-close">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Replace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Replace All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Directory"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionReload"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All_Files"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>F&amp;ind</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_All"/>
    <addaction name="actionReplace"/>
    <addaction name="actionReplace_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuLine_Ending">
     <property name="title">
      <string>&amp;Line Ending</string>
     </property>
     <addaction name="actionWindows"/>
     <addaction name="actionUnix"/>
     <addaction name="actionMac"/>
    </widget>
    <widget class="QMenu" name="menuSyntax">
     <property name="title">
      <string>S&amp;yntax</string>
     </property>
     <addaction name="actionNormal"/>
     <addaction name="actionBash"/>
     <addaction name="actionBatch_File"/>
     <addaction name="actionCMake"/>
     <addaction name="actionCoffeeScript"/>
     <addaction name="actionC_3"/>
     <addaction name="actionC_2"/>
     <addaction name="actionC"/>
     <addaction name="actionCSS"/>
     <addaction name="actionD"/>
     <addaction name="actionFortan"/>
     <addaction name="actionFortan77"/>
     <addaction name="actionDiff"/>
     <addaction name="actionHTML"/>
     <addaction name="actionJava"/>
     <addaction name="actionJavaScript"/>
     <addaction name="actionJSON"/>
     <addaction name="actionLua"/>
     <addaction name="actionMarkDown"/>
     <addaction name="actionMakeFile"/>
     <addaction name="actionMatLab"/>
     <addaction name="actionPascal"/>
     <addaction name="actionPerl"/>
     <addaction name="actionProperties"/>
     <addaction name="actionPython"/>
     <addaction name="actionRuby"/>
     <addaction name="actionSQL"/>
     <addaction name="actionTeX"/>
     <addaction name="actionXML"/>
     <addaction name="actionYAML"/>
    </widget>
    <addaction name="actionSidebar"/>
    <addaction name="actionStatusbar"/>
    <addaction name="actionToolbar"/>
    <addaction name="actionShow_Tabs"/>
    <addaction name="actionShow_Linenumbers"/>
    <addaction name="actionWordWrap"/>
    <addaction name="menuSyntax"/>
    <addaction name="menuLine_Ending"/>
    <addaction name="actionChange_Font"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFind"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>create a new document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open a document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save a document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
   <property name="toolTip">
    <string>Save as </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>C&amp;lose</string>
   </property>
   <property name="toolTip">
    <string>Close document</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="reload">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New &amp;Window</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Window</string>
   </property>
  </action>
  <action name="actionClose_All_Files">
   <property name="text">
    <string>Close All &amp;Files</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>&amp;Replace</string>
   </property>
  </action>
  <action name="actionReplace_All">
   <property name="text">
    <string>R&amp;eplace All</string>
   </property>
  </action>
  <action name="actionSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Sidebar</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;how Statusbar</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sh&amp;ow Toolbar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About </string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionShow_Tabs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sho&amp;w Tabs</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Windows</string>
   </property>
  </action>
  <action name="actionUnix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Unix</string>
   </property>
  </action>
  <action name="actionMac">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Mac</string>
   </property>
  </action>
  <action name="actionFind_All">
   <property name="text">
    <string>F&amp;ind All</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="text">
    <string>&amp;Deselect</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open &amp;Directory</string>
   </property>
  </action>
  <action name="actionBatch_File">
   <property name="text">
    <string>Batch F&amp;ile</string>
   </property>
  </action>
  <action name="actionCoffeeScript">
   <property name="text">
    <string>C&amp;offeeScript</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C#</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="text">
    <string>&amp;Java</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>&amp;HTML</string>
   </property>
  </action>
  <action name="actionJavaScript">
   <property name="text">
    <string>Ja&amp;vaScript</string>
   </property>
  </action>
  <action name="actionJSON">
   <property name="text">
    <string>JSO&amp;N</string>
   </property>
  </action>
  <action name="actionMakeFile">
   <property name="text">
    <string>Ma&amp;keFile</string>
   </property>
  </action>
  <action name="actionMatLab">
   <property name="text">
    <string>MatLab</string>
   </property>
  </action>
  <action name="actionPascal">
   <property name="text">
    <string>&amp;Pascal</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>P&amp;ython</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="text">
    <string>&amp;Bash</string>
   </property>
  </action>
  <action name="actionCMake">
   <property name="text">
    <string>&amp;CMake</string>
   </property>
  </action>
  <action name="actionC_2">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionFortan">
   <property name="text">
    <string>&amp;Fortan</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="text">
    <string>&amp;XML</string>
   </property>
  </action>
  <action name="actionLua">
   <property name="text">
    <string>&amp;Lua</string>
   </property>
  </action>
  <action name="actionMarkDown">
   <property name="text">
    <string>&amp;MarkDown</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="text">
    <string>P&amp;erl</string>
   </property>
  </action>
  <action name="actionRuby">
   <property name="text">
    <string>&amp;Ruby</string>
   </property>
  </action>
  <action name="actionSQL">
   <property name="text">
    <string>&amp;SQL</string>
   </property>
  </action>
  <action name="actionTeX">
   <property name="text">
    <string>&amp;TeX</string>
   </property>
  </action>
  <action name="actionC_3">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionChange_Font">
   <property name="text">
    <string>&amp;Change Font</string>
   </property>
  </action>
  <action name="actionShow_Linenumbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show L&amp;inenumbers</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wo&amp;rdWrap</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>D</string>
   </property>
  </action>
  <action name="actionFortan77">
   <property name="text">
    <string>Fortan77</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="text">
    <string>Diff</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionYAML">
   <property name="text">
    <string>YAML</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
